receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -13,40 +13,85 @@ status:
 # this might need to be unique per sub receiver implementation
 resource_attributes:
   organization.name:
-      enabled: true
-      description: Git Organization or Project Name
-      type: string
+    enabled: true
+    description: Git Organization or Project Name
+    type: string
   git.vendor.name:
-      enabled: true
-      # github, gitlab, bitbucket, gittea
-      description: The name of the Git vendor/provider (ie. GitHub / GitLab)
-      type: string
+    enabled: true
+    # github, gitlab, bitbucket, gittea
+    description: The name of the Git vendor/provider (ie. GitHub / GitLab)
+    type: string
 
-## Attritbutes that will be uncommented when the rest of the metrics are added
+## Attributes that will be uncommented when the rest of the metrics are added
 attributes:
   repository.name:
-      description: The full name of the Git repository
-      type: string
+    description: The full name of the Git repository
+    type: string
+  branch.name:
+    description: The name of the branch in a given repository
+    type: string
 
 metrics:
   git.repository.count:
-      enabled: true
-      description: Number of repositories in an organization
-      unit: 1
-      gauge:
-          value_type: int
-      attributes: []
+    enabled: true
+    description: Number of repositories in an organization
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: []
   git.repository.branch.count:
-      enabled: true
-      description: Number of branches in the repository
-      unit: 1
-      gauge:
-          value_type: int
-      attributes: [repository.name]
+    enabled: true
+    description: Number of branches in the repository
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [repository.name]
   git.repository.contributor.count:
-      enabled: false
-      description: Total number of unique contributors to this repository
-      unit: 1
-      gauge:
-          value_type: int
-      attributes: [repository.name]
+    enabled: false
+    description: Total number of unique contributors to this repository
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [repository.name]
+  git.repository.pull_request.merged.count:
+    enabled: true
+    description: The amount of merged pull requests
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [repository.name]
+  git.repository.pull_request.open.count:
+    enabled: true
+    description: The amount of open pull requests
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [repository.name]
+  git.repository.pull_request.open.time:
+    enabled: true
+    description: The amount of time a PR has been open
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.pull_request.merge.time:
```

Comment:
When I read this, I think: "why would we want to know the age of a pull request that has been merged?"

Reply:
Unless I'm misunderstanding, I believe it's so you can see roughly how long your average PR takes to get merged in a historical view.